### DISCUSSION TITLE
install not working for git LFS

### DISCUSSION THREAD

**Discussion by Aclinashcatnoca** [community/community#139645] [New to GitHub]
I downloaded the git LFS installer and it keeps on popping up with this error message "Could not find Git; can not register Git LFS"

I am unsure of what next steps to take. any help would be greatly appreciated.

**Comment by Sarrac3873** [comment]
> I downloaded the git LFS installer and it keeps on popping up with this error message "Could not find Git; can not register Git LFS"
> 
> I am unsure of what next steps to take. any help would be greatly appreciated.
